app/styles/_section-list.scss

```diff
@@ -0,0 +1,60 @@
+.bc-c-section-list__items {
+  list-style: none;
+  padding-left: 0;
+  @media (min-width: 641px) {
+    padding-left: 30px;
+  }
+}
+
+.bc-c-inset-text {
+  margin-top: nhsuk-spacing(3);
+  max-width: 100%;
+  padding: 0;
+  .bc-c-section-list__item:first-child {
+    border-top: none;
+  }
+  .bc-c-section-list__items {
+    padding-left: 0;
+  }
+}
+
+.bc-c-section-list__item {
+  padding-top: nhsuk-spacing(2);
+  padding-bottom: nhsuk-spacing(2);
+}
+
+.bc-c-section-list__item:after {
+    content: "";
```

Comment:
what is this about?

Reply:
it is because the elements inside it are floated so this class makes sure the div surrounds them. It was already there too!

Reply:
ah ok. I've never come across this before. Can we remove it and see what happens